### PR TITLE
conveted filter-caveats

### PR DESCRIPTION
**Pull request by oussamanahdi**
#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor (non-breaking change which improves implementation)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Non-functional change (xml comments/documentation/etc)

#### Checklist:

- [ ] My code follows the code style of this project.
- [ ] I have read the **CONTRIBUTING** [document](https://github.com/QuantConnect/Lean/blob/master/CONTRIBUTING.md).
- [ ] My branch follows the naming convention `bug-<issue#>-<description>` or `feature-<issue#>-<description>`

### CODE REVIEW

File: 03 Writing Algorithms/12 Universes/04 Equity Options/11 Filter Contracts.php [hunk 1]
```diff
@@ -1,13 +1,13 @@
-<?php include(DOCS_RESOURCES."/universes/option/set-filter.php"); ?>
+<? include(DOCS_RESOURCES."/universes/option/set-filter.php"); ?>
 
 
 <p>The following table describes the filter methods of the <code>OptionFilterUniverse</code> class:</p>
 
-<?php 
+<? 
 echo file_get_contents(DOCS_RESOURCES."/universes/option/option-filter-universe.html");
```
Comment (jaredbroad) [inline 03 Writing Algorithms/12 Universes/04 Equity Options/11 Filter Contracts.php:7]: Please update to include(

File: 03 Writing Algorithms/34 Algorithm Framework/02 Universe Selection/08 Options Universes/61 Option Chained Universe Selection.php [hunk 2]
```diff
@@ -58,13 +58,13 @@
     return option_filter_universe.Strikes(-2, +2).FrontMonth().CallsOnly()</pre>
 </div>
 
-<?php
+<?
+$assetClass("Option");
```
Comment (jaredbroad) [inline 03 Writing Algorithms/34 Algorithm Framework/02 Universe Selection/08 Options Universes/61 Option Chained Universe Selection.php:62]: this is `$assetClass = "Option";` not a method

**Review by jaredbroad — CHANGES_REQUESTED** [review]
Minor bug and include statements

File: 03 Writing Algorithms/34 Algorithm Framework/02 Universe Selection/08 Options Universes/61 Option Chained Universe Selection.php (hunk 2 above)
Comment (oussamanahdi) [reply 03 Writing Algorithms/34 Algorithm Framework/02 Universe Selection/08 Options Universes/61 Option Chained Universe Selection.php]: yes already fixed

**Review by jaredbroad — APPROVED** [review]

**Pull request merged**